core/src/main/java/org/openstack4j/openstack/magnum/MagnumBay.java

```diff
@@ -7,172 +7,273 @@
 import org.openstack4j.openstack.common.GenericLink;
 import org.openstack4j.openstack.common.ListResult;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.MoreObjects;
```

Comment:
https://google.github.io/guava/releases/20.0/api/docs/com/google/common/base/Objects.ToStringHelper.html

Deprecated. 
Use MoreObjects.ToStringHelper instead. This class is scheduled for removal in Guava 21.0.

Objects.ToStringHelper is depreated. I changed it to MoreObjects because we use guava 21 in our project. In guava 21 Objects.ToStringHelper has been removed, so we got NoSuchMethodError in runtime.

Anyway I updated only for guava 20, because guava 21 works with only jdk8. guava 22 supports jdk7 again with 22.0-android version.